crates/tako/src/lib.rs

```diff
@@ -16,8 +26,36 @@ define_id_type!(InstanceId, u32);
 pub type Priority = i32;
 pub type PriorityTuple = (Priority, Priority); // user priority, scheduler priority
 
-pub type Error = crate::common::error::DsError;
+pub type Error = internal::common::error::DsError;
 pub type Result<T> = std::result::Result<T, Error>;
 
-#[cfg(test)]
-pub mod tests;
+pub mod resources {
```

Comment:
These reexports also seem to be duplicated with the `public` directory. Maybe the `public` directory is now useless?